.eslintrc

```diff
@@ -22,6 +22,7 @@
     "no-console": 0,
     "indent": ["error", 4],
     "comma-dangle" : 0,
+    "linebreak-style": 0,
```

Comment:
This is the only important bit!

Reply:
So what does that do, does it disable the ESLint check? Would be good as I'm getting lint errors all over the place.

Reply:
Yep, exactly that :)